pkgs/top-level/all-packages.nix

```
@@ -31428,6 +31428,10 @@ with pkgs;
 
   cups-bjnp = callPackage ../misc/cups/drivers/cups-bjnp { };
 
+  cups-brother-dcp9020cdw-driver = (pkgsi686Linux.callPackage ../misc/cups/drivers/brother/dcp9020cdw { }).driver;
+
+  cups-brother-dcp9020cdw-cups = (pkgsi686Linux.callPackage ../misc/cups/drivers/brother/dcp9020cdw { }).cupswrapper;
```

```suggestion
  dcp9020cdwlpr = (pkgsi686Linux.callPackage ../misc/cups/drivers/brother/dcp9020cdw { }).driver;

  dcp9020cdw-cupswrapper = (callPackage ../misc/cups/drivers/brother/dcp9020cdw { }).cupswrapper;
```
looking at other packages it seems that the cupswrapper doesn't need pkgsi686Linux.callPackage